core/state/journal.go

```diff
@@ -157,34 +166,30 @@ type (
 
 func (ch createObjectChange) revert(s *StateDB) {
 	delete(s.stateObjects, *ch.account)
-	delete(s.stateObjectsDirty, *ch.account)
```

Comment:
Hmm, this looks incorrect, why do we remove this line?

Reply:
field `StateDB.stateObjectsDirty` is removed!